src/queue.ts

```diff
@@ -31,7 +32,7 @@ export class Queue<TItem> extends EventEmitter {
     }).promise()
   }
 
-  startProcessing (handler : (item : TItem) => any | PromiseLike<any>, options?: ProcessOptions): PromiseLike<void> {
+  startProcessing (handler : (item : TItem, message : any) => any | PromiseLike<any>, options?: ProcessOptions): PromiseLike<void> {
```

Comment:
Hmmmm, is this actually a good practice?

If I create an interface for `message`, that means I would have to put every prop and method (and there's a lot). IMO, this is the responsability of the `aws-sdk` package, not our lib's.